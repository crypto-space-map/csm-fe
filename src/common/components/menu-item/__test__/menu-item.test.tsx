import { render } from '@testing-library/react';

import { MenuItem } from '..';

describe('<MenuItem />', () => {
  it('should match snapshot', () => {
    const { container } = render(<MenuItem value="snapshot">snapshot</MenuItem>);
    expect(container).toMatchSnapshot();
  });
  it('should be defined', () => {
    const { container } = render(<MenuItem value="snapshot">defined</MenuItem>);
    expect(container).toBeDefined();
  });
});
