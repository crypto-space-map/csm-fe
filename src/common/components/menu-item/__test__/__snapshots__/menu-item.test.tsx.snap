// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuItem /> should match snapshot 1`] = `
<div>
  <li
    class="MuiMenuItem-root MuiMenuItem-gutters MuiButtonBase-root css-15u9oow-MuiButtonBase-root-MuiMenuItem-root"
    role="menuitem"
    tabindex="-1"
    value="snapshot"
  >
    snapshot
  </li>
</div>
`;
